synce_manager.c

```diff
@@ -0,0 +1,341 @@
+/**
+ * @file synce_manager.c
+ * @brief Interface for managing QL for external clock sources
+ * @note SPDX-FileCopyrightText: Copyright 2022 Intel Corporation
+ * @note SPDX-License-Identifier: GPL-2.0+
+ */
+#define _GNU_SOURCE
+#include <unistd.h>
+#include <pthread.h>
+#include <stdlib.h>
+#include <net/if.h>
+#include <ctype.h>
+#include <errno.h>
+#include "print.h"
+#include "config.h"
+#include "synce_dev.h"
+#include "synce_clock.h"
+#include "synce_manager.h"
+#include "synce_thread_common.h"
+
+struct synce_clock;
+
+enum synce_manager_type {
```

Comment:
the API definitions (lines 23-38 ?) shall be in some external header file, as this would be part of API documentation and shared with the clients

Reply:
moved the shared API definition to synce_external_api.h